docs/README.md

```diff
@@ -78,6 +78,14 @@ docker run --rm `
   jekyll/jekyll `
   jekyll serve
 ```
+> Note: You may seen a warning saying `Auto-regeneration may not work on some Windows versions.` Symptoms of this warning are the inability to automatically see any local website changes in your browser. This makes website changes difficult as you must restart Jekyll after every local change to view the results. To work around this issue, try the `--force-polling` switch to the `jekyll serve` command.
+```powershell
+docker run --rm `
+  --volume="${PWD}:/srv/jekyll" `
+  --publish 4000:4000 `
+  jekyll/jekyll `
```

Comment:
should we specify a tag here? see linux/mac section

Reply:
Fixed